docs/configuration_options.md

```diff
@@ -0,0 +1,275 @@
+
+### branch
+
+**Default** "master"
+
+Name of the checked-out source control branch.
+
+### code_version
+
+A string, up to 40 characters, describing the version of the application code.
+
+Rollbar understands these formats:
+
+ * semantic version (i.e. "2.1.12")
+ * integer (i.e. "45")
+ * git SHA (i.e. "3da541559918a808c2402bba5012f6c60b27661c")
+
+### custom_data_method
+
+The method to call to gather custom data to send with each rollbar request.
+
+```ruby
+def custom_data
+  return {
+    :custom => {
+      :key1 => get_key_one,
+      :key2 => get_key_two
+    },
+    :server => {
+      :root => '/home/username/www/'
+    }
+  }
+end
+```
+
+### default_logger
+
+**Default** `Logger.new(STDERR)`
+
+What logger to use for printing debugging and informational messages during
+operation.
+
+### disable_monkey_patch
+
+**Default** `false`
+
+Disables monkey patching all non-core monkey patches. If you do this you'll
+need to manually `use` an appropriate Rollbar middleware.
+
+Especially useful if you're not using Rails or Sinatra.
+
+### disable_core_monkey_patch
+
+**Default** `false`
+
+Disables our monkey patches in the ruby core. One mandatory monkey patch is left.
+Be careful using this option as it may caused unexpected behavior in some situations.
+
+## Delayed Job Monitoring
+
+### delayed_job_enabled
+
+**Default** `true`
+
+
+Set to false if you have `Delayed`  but do not wish to wrap Delayed jobs with a
+Rollbar notifier.
+
+### report_dj_data
+
+**Default** `true`
+
+Set to `false` to skip automatic reporting of Delayed job data. This can be
+handy if you manually report to Rollbar, or you have another way of catching and
+reporting those errors.
+
+### dj_threshold
+
+**Default** 0 (Report *any* errors)
+
+The number of job failures before reporting the failure to Rollbar.
+
+### failover_handlers
+
+An array of backup handlers if the async handlers fail. Each should respond to
+`call` and should receive a `payload`.
+
+### filepath
+
+For use with `write_to_file`. Indicates location of the rollbar log file being
+tracked by [rollbar-agent](https://github.com/rollbar/rollbar-agent).
+
+### framework
+
+**Default** 'Plain'
+
+Indicates which framework you're using. Common options include 'Rails',
+'Sinatra', and 'Rack' to name a few.
+
+### ignored_person_ids
+
+**Default** `[]`
+
+Ids of people whose reports you wish to ignore. Only works in conjunction with a
+properly defined `person_method` or `person_id_method`.
+
+### logger
+
+The logger to use *instead of* the default logger. Especially useful in `scope`s
+where you wish to send log messages elsewhere.
+
+### payload_options
+
+Extra data to send with the payload.
+
+### person_method
+
+Rails only: A string or symbol giving the name of the method on the controller.
+Should return an object with an `id` method, and optionally `username` and
+`email` methods.
+
+If not using Rails:
+
+Populate the Rack `person_data` key with a hash containing `:id`, and optionally
```

Comment:
Customers need to populate the key `rollbar.person_data` not `person_data` in the Rack environment. It's important emphasize the Rack environment.